app/renderer/components/frame/frame.js

```diff
@@ -591,6 +591,10 @@ class Frame extends React.Component {
           // update the load state; otherwise it will not show the security
           // icon.
           return
+        } else if (!(getBaseUrl === getTargetAboutUrl('about:newtab'))) {
```

Comment:
doesn't this always return true since `getBaseUrl` is a function and `getTargetAboutUrl('about:newtab')` is a string?

Reply:
You are right actually. Sorry for that, I am gonna figure it out asap and update the PR. Okay?